_includes/neurips/2019/presenters.nips.2019.html

```diff
@@ -0,0 +1,146 @@
+<!-- Speakers Grid Section -->
+<section id="speakers" class="bg-light-gray">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-12 text-center">
+                <h2 class="section-heading">Speakers</h2>
+                <h3 class="section-subheading text-muted"></h3>
+            </div>
+        </div>
+        <div class="grid">
+        {% assign featuredItems = site.tags.oral | sort: "featuredOrder" %} 
```

Comment:
tag reference will need to be specific to this workshop and year for example:
 {% assign featuredItems = site.tags.oral-neurips2019 | sort: "featuredOrder" %}

Reply:
I  don't have data for the presenters ; I think they are sending this information so I'm going to wait before I do this change since right now is empty all the presenters info